README.md

```diff
@@ -38,8 +38,8 @@ Please read the Requirements section. Thank you!
 * Improved Boot Screen Support
 
 ## Requirements 
-1. Mac Pro 2008 - 2012 or Xserve 2008 and later
-2. SIP, a.k.a. System Integrity Protection, must be disabled otherwise the patches might not fully function. To do so:
+* Mac Pro 2008 - 2012 or Xserve 2008 or later (It is not recommended to try it out on a supported Mac.)
```

Comment:
Leave this as is. some tasks can be run on a supported Mac. the preinstall and post installs works on a supported Mac and that disk can be cloned.

just BigMac.sh checks right now and those checks may be removed in bigmac2